packages/babel-plugin-transform-runtime/scripts/build-dist.js

```diff
@@ -89,6 +89,21 @@ function writeCoreJS({
 function writeHelpers(runtimeName, { corejs } = {}) {
   writeHelperFiles(runtimeName, { corejs, esm: false });
   writeHelperFiles(runtimeName, { corejs, esm: true });
+  writeHelperExports(runtimeName);
+}
+
+function writeHelperExports(runtimeName) {
```

Comment:
non-esm helpers should also be exported because `package.exports` prioritize over the cjs relative path loading logic, https://github.com/nodejs/node/blob/8a5c7f6abdfb4a8b09bd1de85c6e9819df66ca48/lib/internal/modules/cjs/loader.js#L526